packages/nhsuk-frontend/src/nhsuk/components/radios/radios.mjs

```diff
@@ -26,18 +26,36 @@ export class Radios extends Component {
 
     this.$inputs = $inputs
 
+    this.$inputs.forEach(($input) => {
+      const targetId = $input.getAttribute('aria-controls')
+
+      // Skip radios without aria-controls attributes
+      if (!targetId) {
+        return
+      }
+
+      // Throw if target conditional element does not exist.
+      if (!document.getElementById(targetId)) {
+        throw new ElementError({
+          component: Radios,
+          identifier: `Conditional reveal (\`id="${targetId}"\`)`
+        })
+      }
+    })
+
     // When the page is restored after navigating 'back' in some browsers the
     // state of form controls is not restored until *after* the DOMContentLoaded
     // event is fired, so we need to sync after the pageshow event.
     window.addEventListener('pageshow', () => this.syncAllConditionalReveals())
 
-    // Although we've set up handlers to sync state on the pageshow or
-    // DOMContentLoaded event, init could be called after those events have fired,
-    // for example if they are added to the page dynamically, so sync now too.
+    // Although we've set up handlers to sync state on the pageshow event, init
+    // could be called after those events have fired, for example if they are
+    // added to the page dynamically, so sync now too.
+    this.syncAllConditionalReveals()
 
     // Attach event handler to radioInputs
     this.$inputs.forEach((radioButton) => {
-      radioButton.addEventListener('change', () =>
+      radioButton.addEventListener('click', () =>
```

Comment:
This matching change was missed in https://github.com/nhsuk/nhsuk-frontend/pull/1469/commits/36218777f8b41894f59542775d9177034150f408